### ISSUE TITLE
Option to restore tabs from "Close All Tabs"

### ISSUE THREAD

**Issue by hackermatic** [mozilla-mobile/firefox-ios#15985]
### User Problem
When I use the "Close All Tabs" trash bin -- or more likely, accidentally press it -- I expect to be able to restore the tabs, just as I would any other closed tab or window full of tabs. Today I lost about 100 tabs with no way to recover them (as far as I know, there's no equivalent to grabbing the old tab file on the desktop app).

### Ideas for Solution
If nothing else, a confirmation prompt before closing all the tabs would prevent many such accidents.

**Comment by dnarcese** [comment]
What version of FIrefox are you on? In the current versions we have both a confirmation and a way to undo - curious why you didn't have that option.

**Comment by bcutter** [comment]
Firefox 116.1 (33291)

There is no "undo" option/button/function.

Please assist where to find that one.

Even if there is: how to undo something I as user did not initiate? The update (or the freeze on the first start after the update) killed all open tabs. I guess "undo" is for accidents based on users fat fingers.

This is more a technical issue.

**Comment by dnarcese** [comment]
Yes I am referring only to the Undo that appears after tapping "Close all tabs"

**Comment by bcutter** [comment]
Yeah so, as mentioned: no one affected by this screwed update (still being rolled out by the way) hit that button. It's a bug!

So where (the hell) are my 50+ open tabs please?

Will someone actually start to investigate here? Anything one could provide to help making progress? Let's go!
